mempool/clist_mempool.go

```diff
@@ -231,8 +231,8 @@ func (mem *CListMempool) CheckTxWithInfo(tx types.Tx, cb func(*abci.Response), t
 	// The size of the corresponding amino-encoded TxMessage
 	// can't be larger than the maxMsgSize, otherwise we can't
 	// relay it to peers.
-	if len(tx) > maxTxSize {
-		return ErrTxTooLarge
+	if max := mem.config.MaxMsgBytes - aminoOverheadForTxMessage; len(tx) > max {
```

Comment:
`max := mem.config.MaxMsgBytes - aminoOverheadForTxMessage`

maybe it makes sense to create a func?

Reply:
You are right. I fixed this at https://github.com/tendermint/tendermint/pull/3826/commits/7e9a3318b06be6e26cf12f820a8efb9baf0f7dff.